mkdocs.yml

```diff
@@ -145,14 +145,15 @@ nav:
               - Create Project: user_guides/projects/project/create_project.md
               - Add Members: user_guides/projects/project/add_members.md
           - Python:
+              - Environments overview: user_guides/projects/python/python_env_overview.md
+              - Clone environment: user_guides/projects/python/python_env_clone.md
               - Install Library: user_guides/projects/python/python_install.md
               - Export environment: user_guides/projects/python/python_env_export.md
-              - Recreate environment: user_guides/projects/python/python_env_recreate.md
               - Custom Commands: user_guides/projects/python/custom_commands.md
-              - Python Environment History: user_guides/projects/python/environment_history.md
+              - Environment History: user_guides/projects/python/environment_history.md
           - Jupyter:
-              - Run Spark Notebook: user_guides/projects/jupyter/spark_notebook.md
               - Run Python Notebook: user_guides/projects/jupyter/python_notebook.md
+              - Run PySpark Notebook: user_guides/projects/jupyter/spark_notebook.md
               - Remote Filesystem Driver: user_guides/projects/jupyter/remote_filesystem_driver.md
```

Comment:
Could we make the headings consistent? Some of them uses capital letter for every word, others only for the first word.